### PR TITLE
Adds new sandals.

### PR DESCRIPTION
**Pull request by TheBeoni**
Adds new white sandals into medical and brig phys locker. Adds fancy sandals for NT rep to use. Adds jacksandals and security gas mask into blueshield locker.

http://imgur.com/a/ufZKm

### CODE REVIEW

**Comment by KasparoVy** [comment]
The east and west directions of your fancy sandals are mixed up (facing the opposite direction), just need to switch them. The med sandals are fine

**Comment by TheBeoni** [comment]
Actualy just compiled and tested it and it seems its fine.

**Comment by KasparoVy** [comment]
You're right, my Dream Maker isn't displaying the correct anything. Disregard!

**Comment by KasparoVy** [comment]
[sand.zip](https://github.com/ParadiseSS13/Paradise/files/138970/sand.zip)

Think you could drop those in for the Vox sprites in /icons/mob/species/vox/shoes.dm? Doesn't need a sprite_sheets or species_fit 'cause it's in clothes.dm somewhere

**Comment by KasparoVy** [comment]
Thanks!

**Comment by NTSAM** [comment]
:+1:

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -207,3 +207,18 @@
 	tools = list(/obj/item/weapon/wirecutters)
 
 	time = 40
+
+/obj/item/clothing/shoes/medsandal
```
Comment (Fox-McCloud) [inline code/modules/clothing/shoes/miscellaneous.dm]: why aren't these a subtype of sandals?

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -255,6 +256,8 @@
 		new /obj/item/clothing/shoes/centcom(src)
 		new /obj/item/clothing/accessory/holster(src)
 		new /obj/item/clothing/accessory/blue(src)
+		new /obj/item/clothing/shoes/jackboots/jacksandals(src)
+		new /obj/item/clothing/mask/gas/sechailer(src)
```
Comment (Fox-McCloud) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: Blueshield isn't security; he doesn't need one of these.

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 1 above)
Comment (KasparoVy) [inline code/modules/clothing/shoes/miscellaneous.dm]: Just make it a child like /obj/item/clothing/shoes/sandal/med and /obj/item/clothing/shoes/sandal/fancy and it'll be :moneybag:

**Comment by TheBeoni** [comment]
Fox does that matter? Human Blueshield gets jackboots why not give Unathi and Tajara Blueshield jacksandals?

**Comment by FlattestGuitar** [comment]
The hailer is completely unnecessary and would only encourage Bluecurity, which is something everyone hates.

I think the sandals should stay though, considering there's already a pair of jackboots there for the Blueshield.

**Comment by TheBeoni** [comment]
And what he can do with the mask except for yelling at people? People who play should know better.

**Comment by Aurorablade** [comment]
no they don't actually...i mean i would know better..but i have seen OTHER people...

**Comment by TheBeoni** [comment]
Sec Hailer is gone.

**Comment by FlattestGuitar** [comment]
:+1: for the sandals, still.

Really bad idea to encourage Bluecurity, nice to see it go, to be fair.

**Comment by theColdflame** [comment]
Jacksandals are fine but a secmask is unnecessary and reinforces the notion that blueshield is sec.

**Comment by Fox-McCloud** [comment]
Why is there modifications to the Vox feet sheet?

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 3]
```diff
@@ -207,3 +207,18 @@
...
+/obj/item/clothing/shoes/sandal/white
+	name = "White Sandals"
+	desc = "Medical sandals that nerds wear."
+	icon_state = "medsandal"
+	item_color = "medsandal"
+	species_restricted = null
+
+/obj/item/clothing/shoes/sandal/fancy
+	name = "Fancy Sandals"
+	desc = "FANCY!!."
```
Comment (Fox-McCloud) [inline code/modules/clothing/shoes/miscellaneous.dm]: errant period.

**Comment by TheBeoni** [comment]
So vox can wear those sandals too.

**Comment by Fox-McCloud** [comment]
That's not how it works; those are there for overrides if they don't normally fit.

**Pull request merged**